pulumi-language-dotnet/main.go

```diff
@@ -882,3 +887,145 @@ func (host *dotnetLanguageHost) RunPlugin(
 
 	return nil
 }
+
+func (host *dotnetLanguageHost) Pack(ctx context.Context, req *pulumirpc.PackRequest) (*pulumirpc.PackResponse, error) {
+	// copy the source to the target.
+	projectFile := ""
+	err := filepath.Walk(req.PackageDirectory, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if filepath.Ext(path) == ".csproj" {
+			projectFile = strings.TrimSuffix(path, ".csproj")
+		}
+		return nil
+	})
+
+	if err != nil || projectFile == "" {
+		return nil, fmt.Errorf("find project file: %w", err)
+	}
+
+	build := func() error {
+		err := os.RemoveAll(filepath.Join(req.PackageDirectory, "bin"))
+		if err != nil {
+			return err
+		}
+		err = os.RemoveAll(filepath.Join(req.PackageDirectory, "obj"))
+		if err != nil {
+			return err
+		}
+
+		cmd := exec.Command(host.exec, "build", "-c", "Release")
+		cmd.Dir = req.PackageDirectory
+		return cmd.Run()
+	}
+
+	if err := build(); err != nil {
+		return nil, fmt.Errorf("build error before pack: %w", err)
+	}
+
+	destination := filepath.Join(req.DestinationDirectory, filepath.Base(projectFile))
+
+	cmd := exec.Command(host.exec, "pack", "-c", "Release", "-o", destination)
+	cmd.Dir = req.PackageDirectory
+	cmd.Env = os.Environ()
+
+	if err := cmd.Run(); err != nil {
+		return nil, fmt.Errorf("failed to pack: %w", err)
+	}
+
+	return &pulumirpc.PackResponse{
+		ArtifactPath: destination,
```

Comment:
This should return the actual nuget file

Reply:
Fixed this